ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -11,8 +11,9 @@ import { ObjectExplorerNodeType, StorageKeys } from 'src/app/app.constants'
 import FlatNode from 'src/app/model/schema-object-node'
```

Comment:
This issue already exists in the current master. Since it is not specific to Milestone 4, can we add it in the backlogs list and fix it while clearing backlogs?

Reply:
In that case can we prioritise this backlog item because its a common use case

Reply:
Sure, will clear this first before other backlogs.